modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.scss

```
@@ -0,0 +1,13 @@
+.lfr-objects__object-field-form-base {
+	background-color: #f1f2f5;
```

Please import variable from Clay and use them instead of hardcoded values

```scss
@import 'atlas-variables';
```

```suggestion
	background-color: $light;
```

https://clayui.com/docs/css/color.html